code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -91,6 +91,9 @@
 		for(var/datum/disease/D in M.viruses)
 			if(D.GetDiseaseID() in data)
 				D.cure()
+		if(prob(1) && istype(M, /mob/living/carbon/human))
+			var/mob/living/carbon/human/H = M
+			H.adjustBrainLoss(100)
```

Comment:
Shouldn't this be -100

Everybody knows autists are smarter then the causal neuraltypical.